tests/dummy/app/controllers/application.js

```diff
@@ -6,6 +6,16 @@ export default Ember.Controller.extend({
   buttonStyles: "expand-left expand-right expand-up expand-down contract contract-overlay zoom-in zoom-out slide-left slide-right slide-up slide-down".w(),
   buttonColors: "green red blue mint purple".w(),
   timeout: 2E3,
+  delay: 1000,
+
+
+
+  init(){
+    this._super.apply(this, arguments);
+
+    this.set('selectedAnimation', this.get('buttonStyles').objectAt(0));
+    this.set('selectedColor', this.get('buttonColors').objectAt(0));
```

Comment:
Observers don't get initiated until after `init` is called, so using set just means you're making 2 or 3 extra method calls somewhere for now value.